src/selection.js

```diff
@@ -275,8 +269,8 @@ class Selection {
      * @return  {*}      the new value
      */
     option(name, value) {
-        const options = this.options;
-        return value === void 0 ? options[name] : (options[name] = value);
+        const {options} = this;
+        return value == null ? options[name] : (options[name] = value);
```

Comment:
``x == null`` works the same as ``x === undefined || x === null``, less room for error.